src/ui-shell/sidenav/sidenav-header.component.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, Input } from "@angular/core";
-import { ListItem } from "../../dropdown/dropdown.module";
-import { I18n } from "../../i18n/i18n.module";
+import { ListItem } from "../../dropdown/index";
+import { I18n } from "../../i18n/index";
 
 /**
- * @deprecated Until futher notice. Will be removed in a future version.
+ * @deprecated Until further notice. Will be removed in a future version.
```

Comment:
At what point do we remove this too?

Reply:
we can probably do that now

Reply:
there's a few things that have been waiting, so we should look at removing them now (separate PRs obviously)